strr-host-pm-web/app/stores/document.ts

```diff
@@ -180,14 +180,19 @@ export const useDocumentStore = defineStore('host/document', () => {
     }
   ]
 
-  async function addStoredDocument (doc: File): Promise<void> {
+  async function addStoredDocument (
```

Comment:
addStoreDocument in strata document store does need to be updated with upload step and date?

Reply:
Good point, this one I was not sure. I don't think this endpoint supports it as of today. I think Lekshmi only updated the application update, but not this one. Maybe it would be beneficial to add it in.